docs/api.md

```diff
@@ -475,6 +476,99 @@ Content:
 
 Students are identified by the `client_id` Cookie.
 
+
+### <a name="get-engagement"></a>Get Engagement
+
+Get a lectures engagement values
+
+#### URL
+
+`/lectures/:lecture-id/engagements`
+
+#### Method
+
+GET
+
+#### URL Parameters
+
+##### Optional
+
+* `last=[integer]` 1 or 0. 1 shows only last entry for each student.
+* `recent=[integer]` 1 or 0. 1 shows only recent entries (last 15 min)
+
+#### Success Responses
+
+##### Success
+
+Code: 200
+
+Content:
+```
+{
+    "type": "object",
```

Comment:
This should be the "engagement" array. I know the 'get comments' API is not doing it this way, but it should really be done this way. Just replace this with the content in "engagement".